assets/stylesheets/units/_hr.scss

```diff
@@ -1,5 +1,19 @@
 hr {
-  background-color: tint($nhs-blue, 80%);
+  background-color: $grey-3;
   border: 0;
-  height: 1px;
+  height: $baseline-grid-unit;
+  max-width: 130px;
```

Comment:
If you just used width you wouldn't need the lte(7)... do we think there's a situation where the width will actually need to be fluid at < 130px?